# Admittance parameters for mass damper systems 
spring_matrix: [200,0,0,0,0,0,
                 0,200,0,0,0,0,
                 0,0,200,0,0,0,
                 0,0,0,20,0,0,
                 0,0,0,0,20,0,
                 0,0,0,0,0,20]

mass_matrix: [1,0,0,0,0,0,
              0,1,0,0,0,0,
              0,0,1,0,0,0,
              0,0,0,2,0,0,
              0,0,0,0,2,0,
              0,0,0,0,0,2]

damping_matrix: [2000,0,0,0,0,0,
                 0,2000,0,0,0,0,
                 0,0,2000,0,0,0,
                 0,0,0,200,0,0,
                 0,0,0,0,200,0,
                 0,0,0,0,0,200]

# this will limit the workspace ([x_min x_max y_min y_max z_min z_max])
workspace_limits: [0.45, 1.5, -1.3, 1.3, 0.10, 1.4]

# joint limit equal to 2*pi
joint_limits: [6.283, 6.283, 6.283, 6.283, 6.283, 6.283]

# kinematics manipulator limits
maximum_velocity: [6.05, 6.05, 6.57, 6.57, 6.57, 6.57]
maximum_acceleration: [10.0, 10.0, 10.0, 10.0, 10.0, 10.0]
# maximum_acceleration: [0.20, 0.20, 0.20, 0.20, 0.20, 0.20]

# force sensor dead zone
force_dead_zone: 3.0
torque_dead_zone: 1.0

# admittance control weight
admittance_weight: 2

# inertia reduction bool
inertia_reduction: false

# trajectory velocity parameters
trajectory_vel_scaling: 0.65 # used when feedback
time_scaling_gain: 0.8 # used when create the trajectory

# compliance trajectory parameters 
compliance_trajectory_start_time: 20.0 #ms 
compliance_trajectory_end_time: 200.0 #ms

# end-effector name, which will follow the target pose
EEF_name: TCP